app/src/main/java/io/intrepid/contest/screens/contestcreation/describecontest/DescribeContestFragment.java

```diff
@@ -27,9 +31,35 @@ public DescribeContestPresenter createPresenter(PresenterConfiguration configura
         return new DescribeContestPresenter(this, configuration);
     }
 
+    @OnTextChanged(R.id.contest_description_edittext)
+    public final void onTextChanged(CharSequence newDesription) {
+        if (TextUtils.isEmpty(newDesription)) {
```

Comment:
I agree with Gabi, that will probably mean we can remove onNextInvalidated and onNextValidated too.